custom/icds/__init__.py

```diff
@@ -0,0 +1 @@
+__author__ = 'tyler'
```

Comment:
haha, my pycharm does this too and i still go in and delete it manually every time instead of figuring out how to turn it off...